third-party/realdds/include/realdds/dds-trinsics.h

```diff
@@ -79,6 +98,11 @@ struct motion_intrinsics
 
     rsutils::json to_json() const;
     static motion_intrinsics from_json( rsutils::json const & j );
+
+    bool operator!=( motion_intrinsics const & other ) const
```

Comment:
How come we need `operator ==` for video but `operator !=` for motion?
Probably will need both operators for both types.

Reply:
I just added what I needed, but you're right: I'll add both.

Reply:
Done